msi/createmsi.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+s#!/usr/bin/env python3
```

Comment:
__[Flake8]__ [E262] inline comment should start with '# ' ([link](https://sider.review/gh/repos/19784232/pulls/5769#issue-4409124))

<sub>posted by Sider</sub>